### DISCUSSION TITLE
Question about infura project ID location

### DISCUSSION THREAD

**Discussion by WeTheArtist** [smartcontractkit/full-blockchain-solidity-course-py#652] [General]
I was curious about this web address and tried to find out where it is located but when I try to open it in chrome, it shows 404 error or page not found. Any idea if this is supposed to link to any website?

![image](https://user-images.githubusercontent.com/79281462/147675429-ac92ea5e-220d-496d-a8e7-65d02ed939c2.png)

**Comment by cromewar** [comment]
Hello @WeTheArtist , The endpoint are not meant to be opened as web pages, they are just for API calls, you connect to them using your API key and send request to the server. That's why you cannot access to them as normal links.

**Reply by WeTheArtist** [reply]
Thank you ! That makes sense now :)
